CHANGELOG.md

```diff
@@ -4,12 +4,15 @@
 
 ### Added
 
+- Add Http Header value preprocessor mechanism, that can preprocess defined header value before setting it on the request.
+- Allow user to specify `Autorization` header for Basic Authentication. The value will be converted to Base64,
+  or it will be used is if the valued is starting from `Basic ` prefix.
```

Comment:
starting *with* ? The same in README.md:237

Reply:
Fixed